app/models/candidate_interface/personal_details_form.rb

```diff
@@ -0,0 +1,19 @@
+module CandidateInterface
+  class PersonalDetailsForm
+    include ActiveModel::Model
+
+    attr_accessor :first_name, :last_name, :day, :month, :year, :nationality,
```

Comment:
Should we make this `nationalities`, since we're going to accept multiple at some point?

Reply:
Makes sense

Reply:
We'll rethink this as part of the next PR where we add the second field 👍